README.md

```diff
@@ -41,21 +43,53 @@ generic with minimal need for project-specific behavior.
 The remaining infrastructure is managed internally with TerraForm.
 
 ### 1.3. Project configuration
+Getting started with out attempting to make major contributions, a set of the test suite passes without AWS credentials.
 
-1) Create a Python 3.6 virtualenv and activate it, for example 
+1) Create a Python 3.6 `virtualenv` and activate it, for example 
    
    ```
-   virtualenv .venv
+   cd azul
+   virtualenv -p python3 .venv
    source .venv/bin/activate
    ```
 
-2) Choose a name for your personal deployment. The name should be a short
+1) Setup configuration for dev deployment: 
+   ```
+   cd deployments
+   ln -snf dev .active
+   cd ..
+   ```
+
+2) Load the environment:
+
+    ```
+    source environment
+    ```
+   
+    Examine the output.
+
+3) Install the development prerequisites
+
+    ```
+    pip install -r requirements.dev.txt
+    ```
+
+4) Run `make`. It should say `Looking good!` If one of the sanity checks fails,
+   address the complaint and repeat. The various sanity checks are defined in
+   `common.mk`.
+
+---
+
+#### 1.3.1 For personal deployment (AWS credentials available)
```

Comment:
Add: Creating a personal deployment of Azul allows you test changes on a live system in complete isolation from other users. If you intend to make contributions, this is preferred. You will need IAM user credentials to the AWS account you are deploying to.